content/workshops/fullstack.md

```diff
@@ -0,0 +1,56 @@
+{
+    "type": "tutorial",
+    "layout": "type",
+    "title": "Fullstack Workshop",
+    "linktitle": "fullstack", 
+    "card_icon": "ti-layout",
+    "card_body": "Learn how to create and maintain a fullstack web application",
+    "weight": "1"
+}
+## Who is this workshop for?
+{{<break 1>}}
+This workshop is recommended to developers of all skill levels, that have basic knowledge of fullstack web-development, and are wanting to deploy applications on an AWS server. Cdev is an easy-to-use tool that is focussed on making serverless web-development easily attainable for all creators.  In this workshop we will teach you everthing you need to know to begin building complete web applications on your own.
+{{<break 1>}}
+## Goals of the Workshop
+Create a working fullstack web application using Cdev
+- create a database
+- Create an API
+- Setup routes
+- Test your API using Insomnia.
+- Use your API with a frontend framework (React)
+- Setup Auth0 and user authentication
+
+
+{{<break 1>}}
+
+## Things you should know before joining the workshop
+{{<break 1>}}
+- Python
+- basic knowledge of SQL databases
+- basic knowledge of React
+{{<break 1>}}
+## Required Tools/Programs to Follow Along
+{{<break 1>}}
+- NodeJS/React
+- VS Code
+- Insomnia
+- AWS account
+- Auth0 account
+
+>>> Warning: Amazon’s AWS service may charge you for using its service. Any cost related to the use of an AWS service is your responsibility.

```

Comment:
maybe use a tooltip? or create a snippet with different css because it currently blends into the page